homework.md

```diff
@@ -4,31 +4,31 @@
 
 What is 2 + 2?
 
-5
+4
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+white
 
 ## Question 3
 
 What is the  2 * 3 * 6 * 1 * 10 * 34?
 
-1322
+12240
 
 ## Question 4 
 
 What does HTML stand for?
 
-Hyper Tall Massive Language
+ the set of markup symbols or codes inserted into a file intended for display on the Internet
 
```

Comment:
I believe Stand for refers to the abbreviation.
Therefore, a more appropriate response would be HTML = HyperText Markup Language.